.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: csharp
+
+matrix:
+  include:
+    - os: linux # Ubuntu 14.04
+      dist: trusty
+      sudo: required
+      dotnet: 1.0.0-preview2-003121
+    - os: osx # OSX 10.11
+      osx_image: xcode7.2
+      dotnet: 1.0.0-preview2-003121
+
+script:
```

Comment:
Got it. Maybe then we should run `dotnet --version` as that will not inflate the log? I'm fine either way, whatever is best practice on Travis in general.